Horion/Module/Modules/Fly.cpp

```diff
@@ -1,22 +1,87 @@
 #include "Fly.h"
 
 Fly::Fly() : IModule(0, Category::MOVEMENT, "Enables fly like in creative mode") {
+	mode = (*new SettingEnum(this)).addEntry(EnumEntry("Fly", 1)).addEntry(EnumEntry("CubeGlide", 2));
```

Comment:
I feel bad now because I think I did something like that in TestModule.
Fun fact, in C++, unlike C#, you don't need `new` to initialize a class.